NOTES.md

```diff
@@ -1,3 +1,143 @@
+# Release 2.2.0 (Modern Times)
+
+Released on 2015-11-XX
+
+Release highlights:
+
+* Atomic changefeeds
+* Performance & Scalability improvements
+
+Read the [blog post][2.2-release] for more details.
+
+[2.2-release]: http://rethinkdb.com/blog/2.2-release/
+
+## Compatibility ##
+
+Data files from RethinkDB versions 1.16.0 onward will be automatically migrated.
+As with any major release, back up your data files before performing the upgrade.
+
+If you're upgrading from RethinkDB 1.14.x or 1.15.x, you need to migrate your secondary
+indexes first. You can do this by following these steps:
+* install RethinkDB 2.0.5
+* update the RethinkDB Python driver (`sudo pip install 'rethinkdb<2.1.0'`)
+* run `rethinkdb index-rebuild`
+
+Afterwards, you can install RethinkDB 2.2 and start it on the existing data files.
+
+If you're upgrading directly from RethinkDB 1.13 or earlier, you will need to manually
+upgrade using `rethinkdb dump`.
+
+### API-breaking changes ###
+
+* Changefeeds on `.orderBy.limit` as well as `.get` queries previously provided
+  initial results by default. You now need to pass the `includeInitial: true` optarg to
+  `.changes` to achieve the same behavior.
+* The protocol buffer driver protocol is no longer supported. The JSON-based protocol is
+  now the only supported driver protocol. Certain older drivers (e.g. the Java drivers by
+  @dkhenry and by @npiv) no longer work with RethinkDB 2.2.0. See the [drivers][drivers]
+  list for up-to-date drivers.
```

Comment:
Agreed with Etienne. It would be good to not call out two specific drivers, especially because this implies that there is now no way to use RethinkDB and Java.